config/manager/kustomization.yaml

```diff
@@ -13,7 +13,7 @@ configMapGenerator:
   literals:
   - DEFAULT_BUILDER_RESOURCE_NAME=Dockerfile
   - DEFAULT_KANIKO_SECRET_DEFAULT=regcred
-  - DEFAULT_REGISTRY_REPO=quay.io/kiegroup
+  - DEFAULT_REGISTRY_REPO=registry.kube-system.svc.cluster.local/kiegroup
```

Comment:
This alias is working for pushing but seems not to work for pulling the image.
Try to use the cluster ip of the registry.

If using the registry cluster ip will work both for pushing and pulling the image, I propose simply to document it in a brand new DEV_ON_MINIKUBE md file into a _docs_ folder (to be created in the root of the project).

Reply:
there might be a way to retrieve the registry address internally by querying the k8s api, it will differ from ocp, minikube and k8s.

Reply:
@spolti https://github.com/davidesalerno/kogito-serverless-operator/pull/30/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R181

Reply:
not really, it only takes into account the minikube addon...